libssh2-conduit/ssh-client.hs

```diff
@@ -25,7 +25,7 @@ ssh login host port command = do
       public = home </> ".ssh" </> "id_rsa.pub"
       private = home </> ".ssh" </> "id_rsa"
   withSession host port $ \session -> do
-    r <- checkHost session host port known_hosts
+    r <- checkHost session host port known_hosts [TYPE_MASK]
```

Comment:
I am not sure if this is the right behaviour; it basically allows everything. I opted for this because I did not want to make any more breaking changes.